modules/ROOT/pages/api-extensions.adoc

```diff
@@ -0,0 +1,429 @@
+= Anypoint DataGraph API Extension Reference
+
+The following guide defines API extensions DataGraph. 
+
+
+== Object Type Extensions 
+
+This extension is used to mark that an object type has a primary key, and thus it can be used as reference or extension.
```

Comment:
```suggestion
This extension marks that an object type has a primary key, and thus it can be used as a reference or extension.
```

Reply:
Global suggestion for lead-ins: 

Remove "is used to" as shown in the comment.